### DISCUSSION TITLE
Commits page gives 500 server error

### DISCUSSION THREAD

**Discussion by nakst** [community/community#129408] [Repositories]
### Select Topic Area

Bug

### Body

The commits page on my repository gives a 500 server error. See https://github.com/nakst/gf/commits/master/
Who do I contact to get this fixed?

**Comment by Shubham-Saxena-16** [comment]
Here are some steps to troubleshoot and potentially resolve this issue:

1. Check the Server Logs
If you have access to the server logs, check them for any error messages or stack traces that might provide more information about the cause of the 500 error.

2. Review Recent Changes
Consider any recent changes that might have affected the repository. If you recently pushed new code or made configuration changes, those could be the source of the problem.

3. Inspect Repository Settings
Review the repository settings to ensure there are no misconfigurations that might be causing the issue. This includes checking any webhooks, integrations, or custom scripts that might run on the server.

4. Test Locally
If the repository has scripts or configurations that run on a server, try running them locally to see if you can reproduce the issue. This can help identify if the problem is with the code or the server environment.

5. Clear Cache
Sometimes, server errors can be caused by corrupted cache. Try clearing the cache or restarting the server to see if that resolves the issue.

6. Check GitHub Status
If your repository is hosted on GitHub, check the [GitHub Status](https://www.githubstatus.com/) page to see if there are any ongoing issues that might be affecting your repository.

7. Revert to a Previous Commit
If you recently made changes that might have caused the issue, try reverting to a previous commit to see if the problem persists.

8. Contact Support
If none of the above steps resolve the issue, consider reaching out to the support team of your hosting provider (e.g., GitHub, GitLab) for further assistance. Provide them with as much information as possible, including any error messages from the server logs.

**Comment by nakst** [comment]
It seems like this was fixed? I'll close this support ticket.
